.github/workflows/release.yml

```
@@ -0,0 +1,139 @@
+        with:
+          egress-policy: audit
+
+      - name: Download artifact from build job
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16  # v4.1.8
+        with:
+          name: Windows ${{env.BUILD_CONFIGURATION}} GedValidate
+          path: ${{github.workspace}}
+
+      - name: Get previous tag
+        id: prev_tag
+        run: |
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)